client/app/auth/attachTokens.js

```diff
@@ -1,5 +1,5 @@
 angular.module('distress')
-.factory('AttachTokens', ['$window', '$httpProvider', function ($window, $httpProvider) {
+.factory('AttachTokens', ['$window', function ($window) {
```

Comment:
This was not working properly since the interceptor was not actually intercepting anything. I checked the token in the header and it was undefined as seen by the server so this will fix that issue